testsuite/integration-arquillian/tests/base/pom.xml

```diff
@@ -416,6 +416,65 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>app-server-jetty81</id>
+            <activation>
+                <property>
+                    <name>app.server</name>
+                    <value>jetty</value>
+                </property>
+            </activation>
+            <properties>
+                <app.server>jetty81</app.server> <!--in case the profile is called directly-->
+                <app.server.skip.unpack>false</app.server.skip.unpack>
+            </properties>
+            <dependencies>
+                <dependency>
+                    <groupId>org.keycloak.testsuite</groupId>
+                    <artifactId>integration-arquillian-servers-app-server-jetty-81</artifactId>
+                    <version>${project.version}</version>
+                    <scope>runtime</scope>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.commonjava.maven.plugins</groupId>
+                        <artifactId>directory-maven-plugin</artifactId>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>app-server-jetty94</id>
+            <activation>
+                <property>
+                    <name>app.server</name>
+                    <value>jetty</value>
```

Comment:
the same as above

Reply:
Let's continue the discussion a bit higher in this thread. See https://github.com/keycloak/keycloak/pull/5885#discussion_r257982873